src/event/log_event.rs

```diff
@@ -473,259 +408,6 @@ mod test {
         assert_eq!(json.pointer("/map/map/non-existing"), Some(&fallback));
     }
 
-    #[test]
```

Comment:
These tests are no longer needed as we just proxy to a `vrl::Value` for `LogEvent` instead.

Reply:
Update: re-added these tests in `vrl_target.rs`.